src/main/javascript/software.bytepushers.pick3.lottery.web.TexasPick3UrlScraper.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Created by kalexmills on 7/20/18.
+ */
+var BytePushers = require('bytepushers-js-oop');
+var UrlScraper = require('./software.bytepushers.pick3.lottery.web.UrlScraper');
+
+function TexasPick3UrlScraper(TxPick3UrlScraperConfig) {
+    'use strict';
+    TexasPick3UrlScraper.prototype.superclass.apply(this, [TxPick3UrlScraperConfig]);
+
+    var $ = this.getCheerio();
+
+    this.scrapeTargetUrl = function(targetYear) {
+        var targetUrl = $('#Pick3PastWinningNumbers').find('select > option:contains(' + targetYear + ')').attr("value"),
+            pathArray = this.getUrl().split('/'),
+            baseUrl = pathArray[0] + "//" + pathArray[2];
+
+        if (targetUrl == null) {
+            // TODO: Handle error
+            return null;
```

Comment:
For a date with no corresponding URL, should the scraper return null, or throw an exception?